module/VuFind/sql/mysql.sql

```diff
@@ -409,3 +409,25 @@ CREATE TABLE `access_token` (
   CONSTRAINT `access_token_ibfk_1` FOREIGN KEY (`user_id`) REFERENCES `user` (`id`) ON DELETE CASCADE
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE utf8mb4_unicode_ci;
 /*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `login_token`
+--
+
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8mb4 */;
+CREATE TABLE `login_token` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `user_id` int NOT NULL,
+  `token` varchar(255) NOT NULL,
+  `series` varchar(255) NOT NULL,
+  `last_login` datetime NOT NULL,
+  `browser` varchar(255) NULL,
+  `platform` varchar(255) NULL,
+  `expires` int NOT NULL,
+  `last_session_id` varchar(255) NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `user_id_series` (`user_id`, `series`),
+  KEY `user_id` (`user_id`),
```

Comment:
This still isn't working for me. I think the comma after the KEY line needs to be removed -- adding the missing comma in the line above wasn't the whole solution.